frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -196,22 +196,6 @@ class Dashboard extends Component {
     }
   }
 
-  getDashboardWithFilteredCards = () => {
```

Comment:
Instead of filtering cards based on tab here, this will now be done in `DashboardGrid` so that it also happens in public/embedded dashboards, as they use a different component `PublicDashboard`